deepspeed/module_inject/auto_tp.py

```diff
@@ -119,6 +118,5 @@ def tp_parser(model):
                 gem_list = list(set(gem_list))
                 policy_list = AutoTP.update_policy_list(policy_list, module, gem_list)
                 gem_list = []
-        assert len(policy_list), "AutoTP not supported for model. Please use kernel injection since container policy for model exists." \
-        if AutoTP.kernel_supported(module_list) else "Not able to determine model policy automatically. Please provide policy."
+        assert len(policy_list), "AutoTP not supported for model. Please use kernel injection since container policy for model exists."
```

Comment:
Not all models that fail here have kernel support.

Reply:
kernel support was checked twice. [Here](https://github.com/microsoft/DeepSpeed/blob/49fb30810127764bcad40540f0d0edf905cfc9fe/deepspeed/module_inject/auto_tp.py#L99) & [here](https://github.com/microsoft/DeepSpeed/blob/f1e4fb0b878e48d6fe33dbb2e0969d791d698e86/deepspeed/module_inject/auto_tp.py#L123).

Reply:
Hm, correct. Thanks.